### ISSUE TITLE
Is it not a dual band decoder?

### ISSUE THREAD

**Issue by crlbajsoso** [BarcelonaMedia-Audio/idhoa#1]
Hello, thank you for your sharing. I have some question to trouble you.

1、Is your decoder just for all frequency range? What about dual band decoder?
2、 If I just need 2D planner decoder, just like ITU5.1, can I just set theta = 0?
Thanx

**Comment by davrandom** [comment]
Hi,
idhoa is a code that generates a (one) decoding matrix according to the decoding scheme you choose (basic, maxre, in-phase). So the output of the algorithm is a matrix (an ambisonics decoding), it is not a decoding "engine" (like ambdec is).
The proper way of using this software is:
a) generate the decoding matrices you want to use (idhoa)
b) use them to decode ambisonics with a one/two/three band decoding "engine" (ambdec)

Long story short: running multiple times this software you can generate different matrices and build your own multi-way decoding strategy. If you need help, ask.

For your second question: yes you can put z=0, but I implemented a native 2D version in my fork https://github.com/davrandom/idhoa .

Best,
Davide
ps: a more updated version of this code can be found in my fork https://github.com/davrandom/idhoa
